src/containers/SearchForHelpPage/index.jsx

```diff
@@ -1,24 +1,80 @@
-import React from "react";
+import React, { useState } from "react";
 import SeekHelpMap from "../../components/SeekHelpMap";
 import SeekHelpSingleCard from "../../components/SeekHelpSingleCard";
 import { Container, Row, Col, Dropdown, DropdownButton } from "react-bootstrap";
 import { useTranslation } from "react-i18next";
 import "./index.scss";
 
+const CITIES_WITH_HELP = [
+  {
+    name: "Istanbul",
+    coordinates: {
+      lat: 41.015137,
+      lng: 28.97953,
+    },
+  },
+  {
+    name: "Ankara",
+    coordinates: {
+      lat: 39.92077,
+      lng: 32.85411,
+    },
+  },
+  {
+    name: "Izmir",
+    coordinates: {
+      lat: 38.41885,
+      lng: 27.12872,
+    },
+  },
+];
+
 const SearchForHelpPage = () => {
   const { t } = useTranslation();
+  const [seekHelpCards, setSeekHelpCards] = useState(
+    t("home.seekHelpSection", { returnObjects: true })
+  );
+  const [filteredCity, setFilteredCity] = useState({
+    name: "Istanbul",
+    coordinates: {
+      lat: 41.015137,
+      lng: 28.97953,
+    },
+  });
+
+  const handleSpecialistFilter = (e) => {
+    setSeekHelpCards(() => {
+      if (e === "All") {
```

Comment:
e could be better named here, as i am very surprised to see "e" (which is short for event) being equated to a string